src/main/java/frc/team2412/robot/subsystem/ClimbSubsystem.java

```diff
@@ -45,9 +47,14 @@
 
     private double lastUpdatedTime = Timer.getFPGATimestamp();
 
-    public ClimbSubsystem(WPI_TalonFX motor) {
+    public ClimbSubsystem() {
         setName("ClimbSubsystem");
-        this.motor = motor;
+        this.motor = new WPI_TalonFX(CLIMB_FIXED_MOTOR);
+        /*
+         * climbMotorDynamic = new WPI_TalonFX(CLIMB_DYNAMIC_MOTOR);
+         * climbAngle = new DoubleSolenoid(PneumaticsModuleType.REVPH, CLIMB_ANGLE_UP_SOLENOID,
+         * CLIMB_ANGLE_DOWN_SOLENOID);
+         */
```

Comment:
Delete the dynamic motor & pneumatic, as they aren't being added to the robot

Reply:
Ditto for it's use below

Reply:
Minor detail, but there's still a comment with code for adding dynamic motor to sim.